presto-hive-common/src/main/java/com/facebook/presto/hive/HiveFileContext.java

```diff
@@ -82,7 +83,7 @@ public Optional<Long> getFileSize()
     public void incrementCounter(String name, long value)
```

Comment:
We may need to add `unit` to this function and migrate existing usages.
This can be done in a separate PR.

Reply:
Sounds good. Created an issue to keep track of it - https://github.com/prestodb/presto/issues/17777